File: Arjuna/versions/PR2.0_Alpha_93_04_14/Tests/Local/UidUniqueness/UidUniqueness.cc
/*
 * Copyright (C) 1993
 * 
 * Department of Computing Science,
 * The University,
 * Newcastle upon Tyne,
 * UK.
 */

/*
 * $Id: UidUniqueness.cc,v 1.2 1993/03/18 15:08:03 n048z Exp $
 */

#include <System/iostream.h>
#include <Common/Uid.h>

int main(int, char*[])
{
    Uid uids[16];

    for (int i = 0; i < 16; i++)
	cout << uids[i] << '\n' << flush;
}

File: JTSArjuna/Total-e-Transactions/19072001/JTSArjuna/ArjunaCommon/tests/Uid/UidUniqueness.java
/*
 * Copyright (C) 1998, 1999, 2000, 2001,
 *
 * Arjuna Solutions Limited,
 * Newcastle upon Tyne,
 * Tyne and Wear,
 * UK.  
 *
 * $Id: UidUniqueness.java,v 1.1.36.1 2001/01/24 13:39:35 nmcl Exp $
 */

import com.arjuna.ArjunaCommon.Common.*;

public class UidUniqueness
{
    
public static void main (String[] args)
    {
	for (int i = 0; i < 100; i++)
	{
	    Uid u = new Uid();
	    System.out.println(u + " " + u.hashCode());
	}
    }
    
}
